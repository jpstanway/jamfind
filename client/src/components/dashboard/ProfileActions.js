import React from "react";
import { Link } from "react-router-dom";

const ProfileActions = () => {
  return (
    <div className="row">
      <div className="col-md-12">
        <div className="btn-group btn-group-sm" role="group">
          <Link to="/edit-profile" className="btn btn-custom-actions">
            Edit Profile
          </Link>
          <Link to="/add-experience" className="btn btn-custom-actions">
            Add Experience
          </Link>
          <Link to="/add-education" className="btn btn-custom-actions">
            Add Education
          </Link>
        </div>
      </div>
    </div>
  );
};

export default ProfileActions;
