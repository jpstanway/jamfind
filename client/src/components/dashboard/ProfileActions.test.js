import React from "react";
import { shallow } from "enzyme";
import ProfileActions from "./ProfileActions";

it("renders component", () => {
  expect(shallow(<ProfileActions />)).toMatchSnapshot();
});
