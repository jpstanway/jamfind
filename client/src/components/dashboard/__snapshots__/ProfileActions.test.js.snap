// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ProfileActions />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="col-md-12"
      >
        <div
          className="btn-group btn-group-sm"
          role="group"
        >
          <Link
            className="btn btn-custom-outline-actions"
            replace={false}
            to="/edit-profile"
          >
            Edit Profile
          </Link>
          <Link
            className="btn btn-custom-outline-actions"
            replace={false}
            to="/add-experience"
          >
            Add Experience
          </Link>
          <Link
            className="btn btn-custom-outline-actions"
            replace={false}
            to="/add-education"
          >
            Add Education
          </Link>
        </div>
      </div>,
      "className": "row",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="btn-group btn-group-sm"
          role="group"
        >
          <Link
            className="btn btn-custom-outline-actions"
            replace={false}
            to="/edit-profile"
          >
            Edit Profile
          </Link>
          <Link
            className="btn btn-custom-outline-actions"
            replace={false}
            to="/add-experience"
          >
            Add Experience
          </Link>
          <Link
            className="btn btn-custom-outline-actions"
            replace={false}
            to="/add-education"
          >
            Add Education
          </Link>
        </div>,
        "className": "col-md-12",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/edit-profile"
            >
              Edit Profile
            </Link>,
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/add-experience"
            >
              Add Experience
            </Link>,
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/add-education"
            >
              Add Education
            </Link>,
          ],
          "className": "btn-group btn-group-sm",
          "role": "group",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Edit Profile",
              "className": "btn btn-custom-outline-actions",
              "replace": false,
              "to": "/edit-profile",
            },
            "ref": null,
            "rendered": "Edit Profile",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Add Experience",
              "className": "btn btn-custom-outline-actions",
              "replace": false,
              "to": "/add-experience",
            },
            "ref": null,
            "rendered": "Add Experience",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Add Education",
              "className": "btn btn-custom-outline-actions",
              "replace": false,
              "to": "/add-education",
            },
            "ref": null,
            "rendered": "Add Education",
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="col-md-12"
        >
          <div
            className="btn-group btn-group-sm"
            role="group"
          >
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/edit-profile"
            >
              Edit Profile
            </Link>
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/add-experience"
            >
              Add Experience
            </Link>
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/add-education"
            >
              Add Education
            </Link>
          </div>
        </div>,
        "className": "row",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="btn-group btn-group-sm"
            role="group"
          >
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/edit-profile"
            >
              Edit Profile
            </Link>
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/add-experience"
            >
              Add Experience
            </Link>
            <Link
              className="btn btn-custom-outline-actions"
              replace={false}
              to="/add-education"
            >
              Add Education
            </Link>
          </div>,
          "className": "col-md-12",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Link
                className="btn btn-custom-outline-actions"
                replace={false}
                to="/edit-profile"
              >
                Edit Profile
              </Link>,
              <Link
                className="btn btn-custom-outline-actions"
                replace={false}
                to="/add-experience"
              >
                Add Experience
              </Link>,
              <Link
                className="btn btn-custom-outline-actions"
                replace={false}
                to="/add-education"
              >
                Add Education
              </Link>,
            ],
            "className": "btn-group btn-group-sm",
            "role": "group",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Edit Profile",
                "className": "btn btn-custom-outline-actions",
                "replace": false,
                "to": "/edit-profile",
              },
              "ref": null,
              "rendered": "Edit Profile",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Add Experience",
                "className": "btn btn-custom-outline-actions",
                "replace": false,
                "to": "/add-experience",
              },
              "ref": null,
              "rendered": "Add Experience",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Add Education",
                "className": "btn btn-custom-outline-actions",
                "replace": false,
                "to": "/add-education",
              },
              "ref": null,
              "rendered": "Add Education",
              "type": [Function],
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
